modules/codepipeline/main.tf

```diff
@@ -132,3 +132,7 @@ resource "aws_codepipeline" "codepipeline" {
     }
   }
 }
+
+data "aws_s3_bucket" "state_bucket" {
```

Comment:
This is a reference to the S3 bucket that I was using to store my terraform state. I decided that I did not want to leverage the codepipeline bucket for terraform state.

This bucket information was used later.